UPGRADE.md

```diff
@@ -67,6 +67,20 @@ There is a list of all the repositories maintained by monorepo, changes in log b
     - if you're using Docker-sync, add the directories mentioned in the PR into `sync_exclude` section of your `docker-sync.yml` to make the synchronization faster as well
 - *(optional)* [#557 - php-fpm image has standard workdir (/var/www/html) in ci stage](https://github.com/shopsys/shopsys/pull/557)
     - update your `docker/php-fpm/Dockerfile` and `kubernetes/deployments/webserver-php-fpm.yml` according to [the pull request](https://github.com/shopsys/shopsys/pull/557) to simplify the CI build
+- [#580 Removed trailing whitespaces from markdown files ](https://github.com/shopsys/shopsys/pull/580)
```

Comment:
@TomasLudvik @boris-brtan 
There is a whitespace in `files ]` could you remove it?
This should be definitely `optional`, if it isn't then it doesn't only remove trailing whitespaces